### PR TITLE
Move one more 20H2 queue to server 2022

### LINKED ISSUE #64763: system.globalization.tests.icutests.icushouldbeloaded
|Build|Kind|Start Time|
|---|---|---|
[1589182](https://dev.azure.com/dnceng/public/_build/results?buildId=1589182)|[PR 62740](https://github.com/dotnet/runtime/pull/62740)|2022-03-02|
[1589203](https://dev.azure.com/dnceng/public/_build/results?buildId=1589203)|Rolling|2022-03-02|
[1589450](https://dev.azure.com/dnceng/public/_build/results?buildId=1589450)|[PR 64743](https://github.com/dotnet/runtime/pull/64743)|2022-03-02|
[1589641](https://dev.azure.com/dnceng/public/_build/results?buildId=1589641)|[PR 64567](https://github.com/dotnet/runtime/pull/64567)|2022-03-02|
[1590183](https://dev.azure.com/dnceng/public/_build/results?buildId=1590183)|[PR 64330](https://github.com/dotnet/runtime/pull/64330)|2022-03-02|
[1590727](https://dev.azure.com/dnceng/public/_build/results?buildId=1590727)|Rolling|2022-03-02|
[1590775](https://dev.azure.com/dnceng/public/_build/results?buildId=1590775)|[PR 64759](https://github.com/dotnet/runtime/pull/64759)|2022-03-02|
[1590822](https://dev.azure.com/dnceng/public/_build/results?buildId=1590822)|[PR 64766](https://github.com/dotnet/runtime/pull/64766)|2022-03-02|
[1591380](https://dev.azure.com/dnceng/public/_build/results?buildId=1591380)|[PR 64759](https://github.com/dotnet/runtime/pull/64759)|2022-04-02|
[1591573](https://dev.azure.com/dnceng/public/_build/results?buildId=1591573)|[PR 64766](https://github.com/dotnet/runtime/pull/64766)|2022-04-02|
[1591711](https://dev.azure.com/dnceng/public/_build/results?buildId=1591711)|Rolling|2022-04-02|
[1591751](https://dev.azure.com/dnceng/public/_build/results?buildId=1591751)|[PR 64330](https://github.com/dotnet/runtime/pull/64330)|2022-04-02|
[1591746](https://dev.azure.com/dnceng/public/_build/results?buildId=1591746)|[PR 64800](https://github.com/dotnet/runtime/pull/64800)|2022-04-02|
[1592091](https://dev.azure.com/dnceng/public/_build/results?buildId=1592091)|[PR 64734](https://github.com/dotnet/runtime/pull/64734)|2022-04-02|
[1592314](https://dev.azure.com/dnceng/public/_build/results?buildId=1592314)|[PR 64743](https://github.com/dotnet/runtime/pull/64743)|2022-04-02|
[1592168](https://dev.azure.com/dnceng/public/_build/results?buildId=1592168)|[PR 64800](https://github.com/dotnet/runtime/pull/64800)|2022-04-02|
[1596337](https://dev.azure.com/dnceng/public/_build/results?buildId=1596337)|Rolling|2022-07-02|

### PR DESCRIPTION
**Pull request by sbomer**
Fix one more instance not covered by https://github.com/dotnet/runtime/pull/64827.
This should fix the same failure in runtime-extra-platforms.

Fixes https://github.com/dotnet/runtime/issues/64763

### CODE REVIEW

**Review by safern — APPROVED** [review]
Thanks, don't know how I missed it

**Comment by sbomer** [comment]
@safern @tarekgh could you check the condition I added to IsIcuCompatiblePlatform?

File: src/libraries/System.Globalization/tests/IcuTests.cs [hunk 1]
```diff
@@ -10,8 +10,11 @@ namespace System.Globalization.Tests
     public class IcuTests
     {
         private static bool IsIcuCompatiblePlatform => PlatformDetection.IsNotWindows ||
-                                                       PlatformDetection.IsWindows10Version1903OrGreater;
+                                                       (PlatformDetection.IsWindows10Version1903OrGreater &&
+                                                        // Server core doesn't have icu.dll on SysWOW64
+                                                        !(PlatformDetection.IsWindowsServerCore && PlatformDetection.IsX86Process));
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/64763", typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindows), nameof(PlatformDetection.IsX86Process))]
```
Comment (safern) [inline src/libraries/System.Globalization/tests/IcuTests.cs]: Do we need to include this active issue attribute now that you are including it to the condition that runs the test? 

Also, your PR is taking care of the issue so I wonder if we still need it.

Comment (sbomer) [reply src/libraries/System.Globalization/tests/IcuTests.cs]: Sorry, that was unintentionally included in the commit (from an earlier attempt to fix the original issue). I've removed it already.

**Comment by safern** [comment]
Thanks for taking care of the condition update 😄

**Pull request merged**